### ISSUE TITLE
Add Global Env Variable

### ISSUE THREAD

**Issue by anthropos9** [lando/compose#8]

**Comment by reynoldsalec** [comment]
@anthropos9 yes! But I had to ask the all-knowing Pirog to figure it out...turns out the docs are nested in an old guide for updating from an old RC version of Lando: https://docs.lando.dev/guides/updating-to-rc2.html#global-envvars

So if you add an `appEnv` entry to your `~/.lando/config.yml` it should work.

I'm going to add a reference to `appEnv` in our Global Config docs...it's worth noting that there are obviously _bad_ things that could happen accidentally with this and be super frustrating to debug, so it's something we'll probably discourage this...the preferred route is configuring env variables within a `.lando.yml` file. We're looking at introducing some features that would allow easier sharing of config among team members with administrative granularity, so that might be a future solution that's better.

And huge thanks to Sitecrafting for sponsoring, super cool that your team has supported Lando for so long! LMK if you need more help, going to close this out for now.

**Comment by anthropos9** [comment]
Thanks @reynoldsalec . I actually found another way of getting it to work using the .lando.yml file by adding an override like this 
```
  appserver:
    overrides:
      environment:
        COMPOSER_AUTH: $COMPOSER_AUTH
```

So long as your shell has the `$COMPOSER_AUTH` variable it works. Additionally we only need to add it to the apps/containers that need it. I will experiment with the `appEnv` item anyway for future reference.

We love Lando at SiteCrafting. It makes all our development a zillion times easier.

**Comment by reynoldsalec** [comment]
Ah, your solution is preferable in my mind, I like that.
